### PR TITLE
Update list of contributors for 0.1.48

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update list of contributors for 0.1.48

### CODE REVIEW

File: Contributors.md [hunk 1]
```diff
@@ -129,12 +144,17 @@ The following people have contributed to the SCAP Security Guide project
...
+* Jonathan Sturges <jsturges@jsturges.remote.csb>
 * Philippe Thierry <phil@reseau-libre.net>
+* Derek Thurston <thegrit@gmail.com>
+* tianzhenjia <jiatianzhen@cmss.chinamobile.com>
 * Paul Tittle <ptittle@cmf.nrl.navy.mil>
 * tomas.hudik <tomas.hudik@embedit.cz>
 * Jeb Trayer <jeb.d.trayer@uscg.mil>
 * Matěj Týč <matyc@redhat.com>
 * VadimDor <29509093+VadimDor@users.noreply.github.com>
+* vojtapolasek <krecoun@gmail.com>
```
Comment (jan-cerny) [inline Contributors.md]: This is duplicate of @vojtapolasek :smile:

Comment (yuumasato) [reply Contributors.md]: That is him.
Where is the duplicate in the list?

Comment (yuumasato) [reply Contributors.md]: @jan-cerny I see it, now! Fixed.

**Comment by yuumasato** [comment]
I realized I created a branch in the repo, sorry for that.

**Pull request merged**